src/Php73/Php73.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Symfony\Polyfill\Php73;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php73
+{
+    const NANO_IN_SEC = 1000000000.0;
+
+    /**
+     * @param mixed $var
+     *
+     * @return bool
+     */
+    public static function is_countable($var)
+    {
+        return is_array($var)
+            || $var instanceof \Countable
+            || $var instanceof \ResourceBundle
+            || $var instanceof \SimpleXmlElement;
+    }
+
+    /**
+     * @param bool $asNum
+     *
+     * @return array|float
```

Comment:
So, I confirm this does not work yet: this should be `array|int` and the current logic overflows.
Since hrtime is relative to a random moment in the past, we should just subtract before multiplying, with an initialized offset, which could be computed like:
```php
class Php73
{
    public static $timeOffset;
    // [...]
}

Php73::$timeOffset = time();
```